### ISSUE TITLE
Official doc about flash may be confused

### ISSUE THREAD

**Issue by caorong** [scalatra/scalatra#472]
[flash](http://scalatra.org/2.4/guides/http/flash.html)

doc say，if I set flash in post method and then flash will shown on the "/home" ‘s page。

but actually it will return error. [copyed in the gist](https://gist.github.com/caorong/1cc79e7b88b6edf6befd)

and I think that it should change from 

```scala
get("/home") {
  ssp("/home")
}
```

to 

```scala
get("/home") {
  ssp("/home", "flash" -> flash)
}
```

**Comment by futurechimp** [comment]
Thanks, @caorong,  I've fixed this. Closed by 5a475bf.

**Comment by seratch** [comment]
I've confirmed the fix too. Let me close this issue now.
